plugins/modules/virt_net.py

```diff
@@ -49,17 +49,9 @@
         type: bool
         description:
             - Specify if a given network should be started automatically on system boot.
-    uri:
-        required: False
-        default: "qemu:///system"
-        description:
-            - libvirt connection uri.
-        type: str
-    xml:
-        required: False
-        description:
-            - XML document used with the define command.
-        type: str
+extends_documentation_fragment:
+    - community.libvirt.virt_uri
+    - community.libvirt.virt_xml
 requirements:
     - "python >= 2.6"
     - "python-libvirt"
```

Comment:
This is wrong, it should be libvirt-python

Reply:
we could fix it in a separate PR, thanks for pointing this out

Reply:
I think thats a good idea to make another PR for that